docs/UserGuide.md

```diff
@@ -170,48 +162,65 @@ Format: `add-doctor n/NAME [t/TAG]…​`
 </div><br>
 
 Examples:
-
 * `add-doctor n/Dr Meredith Grey`
-
 * `add-doctor n/Dr Strange t/Dormammu t/IveComeToBargain`
 
 
-### Cleaning all entries in doctor records: `clear-doctor`
+### Clearing all entries in doctor records: `clear-doctor`
 Clears all entries from the doctor records. <br>
-If any appointments exist in the appointment schedule, then this command will not work. 
-`clear-appt` will have to be done first to clear the appointment schedule.
+* Entries in the doctor records can only be cleared if there are no existing appointments in the appointment schedule.
+* Otherwise, `clear-appt` have to called first.
 
 Format: `clear-doctor`
 
 ### Deleting a doctor : `delete-doctor`
-Deletes the specified doctor from the doctor records.
+Deletes the specified doctor from the doctor records.<br>
 
-Format: `delete-doctor INDEX`
+Format: `delete-doctor [--force] INDEX`
```

Comment:
regarding this, im not sure if we should repeat this, since a possible UG bug is `unnecessary repetitions` under `neatness/correctness`:

![image](https://user-images.githubusercontent.com/59121471/113702724-7ac40300-970c-11eb-9584-279031d97cdf.png)

Reply:
Resolved with links from patient commands to doctor commands and the other way round. (bidirectional links)